source/winKernel.py

```diff
@@ -46,6 +46,7 @@ def GetStdHandle(handleID):
 GENERIC_WRITE=0x40000000
 FILE_SHARE_READ=1
 FILE_SHARE_WRITE=2
+FILE_SHARE_DELETE=4

```

Comment:
This one isn't used, but it was the only access right constant that was missing, so people who use winKernel might expect it should be there.